test/files/run/analyzerPlugins.check

```diff
@@ -19,7 +19,7 @@ canAdaptAnnotations(Trees$TypeTree, ?) [7]
 canAdaptAnnotations(Trees$Typed, ?) [3]
 canAdaptAnnotations(Trees$Typed, Any) [1]
 canAdaptAnnotations(Trees$Typed, Int) [1]
-lub(List(Int(1) @testAnn, 2)) [1]
+lub(List(1 @testAnn, 2)) [1]
```

Comment:
This is an extra widen, which seems worst, no?

Reply:
I don't understand what is being printed here. Where is the extra widen?

Reply:
Oh, I misread - it's not `Int(1)` to `Int`...  I dunno what the difference here is, so let's ignore it.  Resolving.